src/ui/pages/Onboarding/Onboarding.tsx

```diff
@@ -65,7 +65,6 @@ const Onboarding = () => {
     },
   ];
 
-  // @TODO - foconnor: This should be op: OperationType when available (non optional)
```

Comment:
Why are we deleting this comment?

Reply:
Done. I updated params of `handleNavigation` . Now we are using `isRecoveryMode` boolean flag instead of optional string